tests/integration/test_logs.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+import pytest
+
+from tests.integration import utility_test_functions
+
+workflows_path = Path(__file__).parents[1].joinpath("workflows/")
+
+
+class TestLogs:
+    @pytest.mark.parametrize(
+        "wdl, json_file, expected_logs",
+        [
+            (
+                "tests/workflows/helloWorld.wdl",
+                "tests/workflows/helloWorld.json",
+                "No logs with status ['Failed'] found for workflow, try adding the argument '-s ALL' to list logs with any status\n",
+            ),
+            (
+                "tests/workflows/helloWorldFail.wdl",
+                "tests/workflows/helloWorld.json",
+                "HelloWorld.HelloWorldTask:\tFailed\t Backend Logs Not Found\n",
+            ),
+        ],
+    )
+    def test_logs(
+        self,
+        local_cromwell_url: str,
+        wdl: str,
+        json_file: str,
+        expected_logs: str,
+        ansi_escape,
+    ):
+        # submit workflow
+        test_workflow_id = utility_test_functions.submit_workflow(
```

Comment:
I wonder if we could refactor out another utility function for integration tests that is `submit-wait-runcommand`.